OpenEdXMobile/src/main/java/org/edx/mobile/view/MyCoursesListFragment.kt

```diff
@@ -60,6 +72,7 @@ class MyCoursesListFragment : OfflineSupportBaseFragment(), RefreshListener {
     private var refreshOnResume = false
     private var refreshOnPurchase = false
     private var lastClickTime: Long = 0
+    private var unfulfilledPurchases: List<Pair<String, String>> = emptyList()
```

Comment:
Please Add a comment on temporary methods and variables.

Reply:
Not a temporary variable.